chia/policy/fee_estimator_zero.py

```diff
@@ -0,0 +1,20 @@
+from chia.policy.fee_estimation import FeeBlockInfo, FeeMempoolInfo
+from chia.policy.fee_estimator import FeeEstimatorConfig
+from chia.util.ints import uint64
+
+
+class FeeEstimatorZero:  # FeeEstimatorInterface Protocol
```

Comment:
Missing `mempool_size()` and `mempool_max_size()` methods? Not sure it matters for this implementation.

Reply:
Yes, I'll remove this demo estimator in the final patch.